packages/form-js-viewer/assets/form-js.css

```diff
@@ -142,11 +170,71 @@
   width: 100%;
   padding: 8px;
   margin: 4px 0;
-  border-width: 1px;
-  border-style: solid;
+  border: var(--border-definition);
+  border-radius: 3px;
+}
+
+.fjs-container .fjs-input-group {
+  display: flex;
+  width: 100%;
+  margin: 4px, 0;
+  border: var(--border-definition);
+  border-radius: 3px;
+}
+
+.fjs-container .fjs-input-group.disabled,
+.fjs-container .fjs-input-group.disabled .fjs-input,
+.fjs-container .fjs-input-group.disabled .fjs-input-adornment {
+  border-color: var(--color-borders-disabled);
+}
+
+.fjs-container .fjs-taglist .fjs-taglist-input,
+.fjs-container .fjs-input-group .fjs-input {
+  border: none;
+  border-radius: 0;
+  margin: 0;
+  outline: 0;
+}
+
+.fjs-container .fjs-input-group .fjs-input-adornment {
+  border-width: 0;
+}
+
+.fjs-container .fjs-input-group .fjs-input-adornment.border-left {
+  border-left-width: 1px;
+}
+
+.fjs-container .fjs-input-group .fjs-input-adornment.border-right {
+  border-right-width: 1px;
+}
+
+.fjs-container .fjs-input-group .fjs-input-adornment.border-radius-right {
+  border-radius: 0 3px 3px 0;
+}
+
+.fjs-container .fjs-input-group .fjs-input-adornment.border-radius-left {
+  border-radius: 3px 0 0 3px
+}
+
+.fjs-container .fjs-input-group .fjs-input,
+.fjs-container .fjs-input-group.disabled .fjs-input {
   border-radius: 3px;
 }
 
+.fjs-container .fjs-vertical-group {
+  display: flex;
+  width: 100%;
+}
+
+.fjs-container .flatpickr-wrapper {
+  width: 100%;
+}
+
+.fjs-container .flatpickr-calendar.hasTime.noCalendar {
+  width: auto !important;
+  max-width: 250px !important;
+}
+
```

Comment:
Shouldn't we move that to the bottom part (flatpickr overrides)

Reply:
Done.